packages/mui-base/src/useOption/useOption.types.ts

```diff
@@ -4,7 +4,7 @@ export interface SelectOption<Value> {
   value: Value;
   label: React.ReactNode;
   disabled?: boolean;
-  ref: React.RefObject<Element>;
+  ref: React.RefObject<HTMLElement>;
```

Comment:
Is there anything that specifically requires HTMLElement?

Reply:
`useList`'s `getItemDomElement` expects an `HTMLElement`: https://github.com/mui/material-ui/blob/master/packages/mui-base/src/useList/useList.types.ts#L117